agent/php_execute.c

```diff
@@ -1715,48 +1823,143 @@ static void nr_php_instrument_func_begin(NR_EXECUTE_PROTO) {
    */
   if ((NULL != wraprec) || (NRINI(tt_detail) && NR_OP_ARRAY->function_name)) {
     /*
```

Comment:
Since this case results in just a `return` if false, we can refactor it to return immediately and alleviate some of the `if` nesting inside it.
This should do the trick:
```c
  if ((NULL == wraprec)
      && (!NRINI(tt_detail) || NULL == NR_OP_ARRAY->function_name)) {
    return;
  }
```
Along with removing the `}` at the end of the function.

Reply:
Please see: 
[fed7f9c](https://github.com/newrelic/newrelic-php-agent/pull/580/commits/fed7f9c6c7a980c3d8bb1d87f43f9218363fc105)

and 

[4c64593](https://github.com/newrelic/newrelic-php-agent/pull/580/commits/4c645933f57d20c7395153e36b409cc2358b0463)